### DISCUSSION TITLE
Add chips like arduino & pi

### DISCUSSION THREAD

**Discussion by RakibRyan** [logisim-evolution/logisim-evolution#1596] [General]
It's a very good app. Unfortunately no library for micro controllers. And maybe an area to program these micro controllers.

**Comment by maehne** [comment]
We already offer instruction set simulators for Nios2s and Risc V in the "System On a Chip" built-in library (which needs to be activated via `Project -> Load Library -> Built-in Library...`). Though, these components lack documentation, because we use them primarily in one course at BFH. So, it is feasible, but requires considerable effort to add more processor architectures for which we lack resources. Also, one needs to precisely define the scope for such an enhancement. Otherwise, we risk that Logisim-evolution looses its focus and thus attractiveness as a teaching tool.
